src/components/v2/ApyChart/formatToReadableDate.ts

```diff
@@ -0,0 +1,6 @@
+import moment from 'moment';
```

Comment:
`moment` isn't recommended anymore. We should look at using `luxon` or `date-fns`

Reply:
Works for me 👍